test/binary/binary-boundary.test.py

```diff
@@ -10,13 +10,15 @@
 from internal.memcached_connection import MemcachedBinaryConnection
 from internal.memcached_connection import STATUS, COMMANDS
 
+from internal.compat import string_types
+
```

Comment:
Sorry for nitpicking, but it looks irrelevant to the 'test: convert print to function' commit.

Reply:
Right, fixed. Thanks!